coalib/settings/FunctionMetadata.py

```diff
@@ -140,11 +140,13 @@ def return_arg(x):
         if annotation is None:
             annotation = return_arg
 
+        lines = section._get_lines(param)
         try:
             return annotation(section[param])
         except (TypeError, ValueError):
             raise ValueError('Unable to convert parameter {!r} into type '
-                             '{}.'.format(param, annotation))
+                             '{}.'.format(param, annotation) +
+                             '\n.coafile lines: ' + str(lines))
```

Comment:
This is the one usage of the line numbers which you might like to keep in the patch, as it is in `coalib/settings`, but I suggest that you save it in a backup branch, and remove it until you have good tests cases all passing and getting positive reviews of your test cases.